src/components/Footer.jsx

```diff
@@ -21,14 +21,28 @@ const Footer = () => {
         <p>All Rights Reserved.</p>
       </div>
       <div className="flex flex-col items-end">
-        <div>
-          sales@jmproducts.com <FaInstagram className=" inline" />
-        </div>
-        <div>
-          +1-800-624-2198 <FaLinkedin className=" inline" />
-        </div>
-        <div>
-          +1-818-837-0205 <AiFillFacebook className=" inline" />
+        <div className="flex flex-col justify-end gap-1">
+          {data.map((item, index) => (
+            <div
+              key={index}
+              className="flex flex-row gap-2 justify-end items-center"
+            >
+              {index === 0 && (
+                <a className="hover:underline" href={item.contact}>
+                  {item.contact}
+                </a>
+              )}
+              {index > 0 && <p>{item.contact}</p>}
```

Comment:
This is valid but it can probably be simplified down to a simple ternary statement. 

Something like `{index === 0 : the link ? no link}`

Also use the `Link` component from next instead of the `a` tag